builder/amazon/chroot/step_register_ami.go

```diff
@@ -18,7 +18,7 @@ type StepRegisterAMI struct {
 	EnableAMISriovNetSupport bool
 }
 
-func (s *StepRegisterAMI) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
+func (s *StepRegisterAMI) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
```

Comment:
Why rename this, given that `ctx` remains unused?

Reply:
It's a WIP; I'll be adding context to all of these.